### PR TITLE
Added test documentation link to wpt's README.md

### LINKED ISSUE #15603: Add test documentation link to wpt's README.md
For the sake of convenience, the documentation for writing web platform tests can be provided in the `wpt` README file. (http://testthewebforward.org/docs/testharness-library.html)

### PR DESCRIPTION
**Pull request by 0xRampey**
For the sake of convenience, the documentation for writing web platform tests can be provided in the wpt README file.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix #15603  (github issue number if applicable).

- [ ] There are tests for these changes OR
- [X] These changes do not require tests because of documentation change.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/15608)

### CODE REVIEW

**Comment by SimonSapin** [comment]
Looks good, except that this implies that every WPT test uses testharness.js, while some don’t since they’re reftests. This change should mention reftests, but I’m not sure where to link to describe them. @jgraham, do you have a suggestion?

File: tests/wpt/README.md [hunk 1]
```diff
@@ -128,6 +128,8 @@ be run in the following manner like any other WPT test:
     ./mach test-wpt tests/wpt/path/to/new/test.html
     ./mach test-wpt tests/wpt/path/to/new/reftest.html
 
+The tests are written using [testharness.js](https://github.com/w3c/testharness.js/). Documentation can be found [here](http://testthewebforward.org/docs/testharness-library.html).
```
Comment (Ms2ger) [inline tests/wpt/README.md]: Maybe link to http://web-platform-tests.org/writing-tests/index.html

Comment (prampey) [inline tests/wpt/README.md]: @Ms2ger Perhaps it is more apt for your link to be at the top of the page to serve as a primer for writing tests in general. 
We can get more specific by setting a link for reftests after its explanation ends and the same for testharness.js tests.

**Comment by SimonSapin** [comment]
Looks good, thanks!

@bors-servo r+

**Comment by prampey** [comment]
You're welcome :)

**Pull request merged**